react/components/SKUSelector/utils/index.ts

```diff
@@ -51,7 +51,13 @@ export const isColor = (variation: string) => {
   if (!variation) return false
 
```

Comment:
can you save `const variationLowerCase = variation.toLowerCase()` here and then use it to compare with all the others?